client/src/modules/Admin/Components/Admin.tsx

```diff
@@ -235,6 +237,24 @@ export const Admin = () => {
     })
   }
 
+  /**
+   * Call when user selects "Update Descriptions" button. Scrapes the Course API 
+   * to retrieve course description and stores them in the Course database.
+  */
+  function updateDescriptions() {
+    console.log('Updating course descriptions')
+    setDisableInit(true)
+    axios.post('/api/admin/courseDescriptions/add', { token: token }).then((response) => {
```

Comment:
In the future, when the branch updating route names is merged, we could slightly update this to something like '/api/admin/course/desc' for consistency. Not sure about the example I provided but we can model it off the other ones :)